care/settings/base.py

```diff
@@ -346,3 +346,5 @@ class Settings(LoggerSettingsMixin, Configuration):
             }
         }
     }
+
+    WEBAPP_BASE_URL = 'http://localhost:3000'
```

Comment:
put it in local.py

Reply:
will override in prod.py. Need some default value here.